### PR TITLE
Add event notes ("minutes")

### PR DESCRIPTION
**Pull request by ThiefMaster**
Some weird legacy meeting styles haven't been updated yet and compiling minutes is still missing, but everything else is ready - so some reviewing would be great. Then we can put it into master already (since wip/attachments is based on it anyway).

### CODE REVIEW

File: indico/htdocs/js/indico/jquery/ajaxdialog.js [hunk 1]
```diff
@@ -75,12 +78,19 @@
             }
         });
 
+        function confirmClose() {
+            var forms = popup.contentContainer.find('form');
+            var hasChanges = forms.length && forms.filter(function() {
+                return $(this).data('fieldsChanged');
+            }).length;
+            var message = $T('You have unsaved changes. Do you really want to close the dialog without saving?');
```
Comment (nop33) [inline indico/htdocs/js/indico/jquery/ajaxdialog.js]: :+1:

File: indico/modules/events/util.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import request
+
+
+def get_object_from_args(args=None):
```
Comment (nop33) [inline indico/modules/events/util.py]: Very useful! :+1:

File: indico/web/util.py [hunk 3]
```diff
@@ -30,6 +30,31 @@ def inject_js(js):
     g.injected_js.append(Markup(js))
 
 
+def jsonify_template(template, _render_func=render_template, **context):
```
Comment (nop33) [inline indico/web/util.py]: When should we use this method? When loading templates with AJAX?

Comment (ThiefMaster) [inline indico/web/util.py]: Usually together with ajaxDialog (the comment in ajaxDialog.js mentions this function)

Comment (nop33) [inline indico/web/util.py]: yy saw it, thx.

**Pull request merged**